src/css/welcome-page.css

```diff
@@ -1,35 +1,39 @@
-@import url('https://fonts.googleapis.com/css?family=Condiment&display=swap');
-@import url('https://fonts.googleapis.com/css?family=Abril+Fatface&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Lato&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Raleway&display=swap');
 
 
 #welcome h1{
-  text-align: center;
-  font-size: 5rem;
+  font-size: 80px;
   line-height:1.6;
-  color:whitesmoke;
-  font-family: 'Condiment', cursive;
-  text-align: center;
+  color:rgba(0, 0, 0, 0.808);
+  font-family: 'Lato', sans-serif !important;
+  font-weight: lighter !important;
   padding-bottom: 5rem;
+  margin-right: 700px;
+  margin-top: -120px;
 }
 
 #welcome h5{
-  text-align: center;
   font-size: 2rem;
   margin-top: -70px;
```

Comment:
Becca, instead of using the margins, you guys can do what I showed Mieyser before on her old welcome-page PR where you wrap the text in a div and use:
position: relative;
top: 50%;
left: 50%;
transform: translate(-50%, -50%)